src/main/java/io/vertx/core/http/impl/Http2ServerStream.java

```diff
@@ -128,4 +128,13 @@ void handleClose() {
       }
     }
   }
+
+  void hasBeenRoutedTo(String routeName) {
```

Comment:
rename it `routed` like the original method.

Reply:
I had it that way but it created an overload issue when not also returning the same type, which is not really possible here. Ideas?

Reply:
Done.